src/Symfony/Bundle/WebProfilerBundle/DependencyInjection/Configuration.php

```diff
@@ -38,7 +38,7 @@ public function getConfigTreeBuilder()
             ->children()
                 ->booleanNode('toolbar')->defaultFalse()->end()
                 ->booleanNode('intercept_redirects')->defaultFalse()->end()
-                ->scalarNode('excluded_ajax_paths')->defaultValue('^/(app(_[\\w]+)?\\.php/)?_wdt')->end()
+                ->scalarNode('excluded_ajax_paths')->defaultValue('index.php?_wdt')->end()
```

Comment:
that's not the right update. It will now match `index.php_wdt` and `index.ph_wdt` (and anywhere in the URL, not only at the start), while it should be matching `/_wdt` and `/index.php/_wdt`